src/Sarif.Viewer.VisualStudio/Guids.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using System;
+
+namespace SarifViewer
+{
+    internal sealed partial class Guids
+    {
+        public const string guidVSPackageString = "b97edb99-282e-444c-8f53-7de237f2ec5e";
```

Comment:
that's fine. fyi, legacy VS is camel-cased and emitted vsix follows this pattern. changed it, however.